engine/consensus/matching/engine.go

```diff
@@ -1168,11 +1182,6 @@ func (e *Engine) requestPendingReceipts() (int, uint64, error) {
 		// check if we have an result for the block at this height
 		blockID := header.ID()
 
-		_, ok := knownResultForBlock[blockID]
-		if ok {
-			continue
-		}
-
 		// since the index is only added when the block which includes the receipts
 		// get finalized, so the returned receipts must be from finalized blocks.
 		// Therefore, the return receipts must be incoporated receipts, which
```

Comment:
Outdated comment I believe, we now index on block incorporated

Reply:
updated comment in https://github.com/onflow/flow-go/pull/444/commits/c4ea8726dff06c912a3e6aeee2128b526421a135